src/mk/def_rules.mk

```diff
@@ -387,6 +387,10 @@ SQLCBLIPGM_OBJTYPE := *PGM
 SQLCBLIPGM_OPTION := $(OPTION)
 SQLCBLIPGM_TGTRLS := $(TGTRLS)
 
+SQLPRC_TGTRLS := $(TGTRLS)
+
+SQLTRG_TGTRLS := $(TGTRLS)
+
```

Comment:
so why specifically for SQLPRC and SQLTRG, what about all the others.  Why not have a generic SQL_TGTRLS?
Does the customer use TGTRLS or SQLPRC_TGTRLS in their Rules.mk?

Reply:
https://github.com/IBM/ibmi-bob/blob/fd101c793e391c212eca0f926fa017e14c804a63/src/mk/def_rules.mk#L388
https://github.com/IBM/ibmi-bob/blob/fd101c793e391c212eca0f926fa017e14c804a63/src/mk/def_rules.mk#L382

If you notice, we previously have specific TGTRLS for our different types of SQL. This change was made to follow our previous format.

The customer did not specify anything to do with TGTRLS, but COMMIT.

However, to fix the bug we had switched to recursive assignment, which exposed that these 2 types were being passed in `UNKNOWN_FILE_TYPE` due to the resolution of `programTGTRLS`

Reply:
As you can see from here https://ibm.github.io/ibmi-bob/#/welcome/features?id=support-sql-pseudo-source
there are a few more extensions that we should also support such as SQALIAS, SQLSEQ, SQLUDF, SQLUDT